xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-ui/pom.xml

```diff
@@ -24,20 +24,20 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.xwiki.platform</groupId>
-    <artifactId>xwiki-platform-image</artifactId>
+    <artifactId>xwiki-platform-image-lightbox</artifactId>
     <version>14.1-SNAPSHOT</version>
   </parent>
-  <artifactId>xwiki-platform-image-ui</artifactId>
-  <name>XWiki Platform - Image - UI</name>
+  <artifactId>xwiki-platform-image-lightbox-ui</artifactId>
+  <name>XWiki Platform - Image - Lightbox - UI</name>
```

Comment:
missing a description I think. Proposal: `Display images from the page content in a lightbox`